curated_transformers/models/types.py

```diff
@@ -48,6 +48,6 @@
 TransformerBackpropT = Callable[[List[List[Floats2d]]], Any]
 TransformerModelT = Model[TransformerInT, TransformerOutT]
 
-ScalarWeightInT = List[Ragged]
-ScalarWeightOutT = Ragged
+ScalarWeightInT = List[List[Ragged]]
```

Comment:
We didn't really do this for other types, but maybe we should start adding small comments what the lists are here? 

Not sure what good notation would be?

I just want to make sure that if someone reads the types knows what the input to e.g. scalar weighting actually means. (docs -> layers -> representations vs. layers -> docs -> representations).

Reply:
Good idea. `something -> something else` looks decently readable to me.